refdb-sql/postgres/pgpass.conf

```diff
@@ -0,0 +1 @@
+postgres:5432:postgres:root:root
```

Comment:
This file should be taken out of source control, can create one dynamically as part of CI or deployment if needed.